src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/NoPdfFileFoundException.java

```diff
@@ -1,10 +1,36 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.exceptions;
 
-public class NoPdfFileFoundException extends RuntimeException {
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Event;
+
+public class NoPdfFileFoundException extends RuntimeException implements EventRelatedThrowable {
 
     private static final long serialVersionUID = 9143161748679833084L;
 
-    public NoPdfFileFoundException(String message) {
-        super(message);
+    private static final Event EVENT = Event.DOC_FAILURE;
```

Comment:
I'd also rename this to DOC_FAILURE_EVENT. It really reads better without any need to navigate to the definition